app/models/inbox.rb

```diff
@@ -50,6 +50,7 @@ class Inbox < ApplicationRecord
                                                        message: I18n.t('errors.inboxes.validations.name') }
   validates :account_id, presence: true
   validates :timezone, inclusion: { in: TZInfo::Timezone.all_identifiers }
+  validates :out_of_office_message, length: { maximum: 10_000 }
```

Comment:
@LiamAshdown I would suggest putting this in the `limits.rb` file so that there is consistency.

Reply:
Done :)